Jenkinsfile

```diff
@@ -15,6 +15,7 @@ node('ibm-jenkins-slave-nvm') {
 
   def pipeline = lib.pipelines.generic.GenericPipeline.new(this)
   def manifest
+  def zowePaxUploaded
```

Comment:
Current way of uploading artifacts works fine, but I found for each build, it appears 2-3 builds here https://zowe.jfrog.io/zowe/webapp/#/builds/zowe-install-packaging%20::%20PR-1763/25/1604121016053. It's because we call `uploadArtifacts` several times. If we want to make it better, can define

```
def zoweArtifacts = []
```

Then at the end of packaging operation, assign it with:

```
zoweArtifacts = [
      '.pax/zowe.pax',
      '.pax/zowe-smpe.zip',
      '.pax/smpe-promote.tar',
      '.pax/pd.htm',
      '.pax/smpe-build-logs.pax.Z',
      '.pax/AZWE*'
    ]
```

Then change docker `uploadArtifacts` to `zoweArtifacts.add('zowe-v1-lts.s390x.tar')` and `zoweArtifacts.add('zowe-v1-lts.amd64.tar')`.

Last move publish stage after docker build stages and change to:

```
pipeline.publish(artifacts: zoweArtifacts)
```

This is optional. Or let me know if you think it's good and would like me to add a commit.